gamescope-session-playtron/sources

```diff
@@ -1 +0,0 @@
-SHA512 (gamescope-session-playtron-0.2.0.tar.gz) = 679bc990568067f8766b8f2e5f4d08270e9afaeaa183aa1247ff5c41d9c4c6ef9fbd451f4245adfd65f1d8478a7a8040bd921ba313a09d34fd4888887da7517d
```

Comment:
Does not fail the package build when incorrect. Drop it and save on wasted effort.

Reply:
I revisited this recently. The `sources` file is checked by `fedpkg` when building official Fedora packages. For example, this is used for our Linux kernel builds which are forked from Fedora. It is unused outside of that use-case. We can drop it for now but I would like to revisit this one day. We need to add in our own logic for checking hashes.